### DISCUSSION TITLE
Embodied interaction with mediapipe

### DISCUSSION THREAD

**Discussion by NahuelGerthVK** [google-ai-edge/mediapipe-samples#265] [Show and tell]
Hey mediapipe team,

I just wanted to thank you for this awesome tool and share some work I made in the field of embodied interaction:

**Face painter**
- draw on a canvas with your mouth
- I am testing this for children and elderly people as a form of inclusive digital media
- https://nahuelgerth.de/lab/face-painter

![image](https://github.com/googlesamples/mediapipe/assets/35173372/66aa392e-f065-4307-aac6-1f05c52250e2)

**Embodied typography**
- manipulate letters with your body
- how can lettershapes be manipulated, drawn and generated through bodily gestures?
- https://nahuelgerth.de/lab/36-days-of-type

![230418_hands_letter_i_4_small_thumb](https://github.com/googlesamples/mediapipe/assets/35173372/33d6deaf-fbce-44c2-96eb-9a567ab560ae)

Looking forward to test all the new stuff you added lately.

Thank you and best wishes
Nahuel

**Comment by nstafford** [comment]
Gruess dich Nahuel.  This project is so cool.  I have just started playing with mediapipe in python and it really is fun to use your face as an input device.  I am using it as a game input device.

**Reply by LearningnRunning** [reply]
Can you come take a look at mine??
https://github.com/googlesamples/mediapipe/discussions/342
